bridges/SteamCommunityBridge.php

```diff
@@ -124,4 +130,62 @@ public function collectData() {
 				break;
 		}
 	}
+
+	public function collectWorkshop() {
+		$category = $this->getInput('category');
+		$html = $this->getMainPage();
+		$workShopItems = $html->find('div.workshopItem');
+
+		foreach($workShopItems as $workShopItem) {
+			$author = $workShopItem->find('div.workshopItemAuthorName', 0)->find('a', 0);
+			$author = strip_tags($author);
```

Comment:
This should also work.

```suggestion
			$author = $author->innertext;
```